Tests/UnitTests/Plugins/Geant4/Geant4ConvertersTests.cpp

```diff
@@ -63,7 +64,8 @@ BOOST_AUTO_TEST_CASE(Geant4AlgebraConversion) {
 }
 
 BOOST_AUTO_TEST_CASE(Geant4CylinderConversion) {
-  G4Tubs cylinder("Cylinder", 399., 401., 800., 0., 2 * M_PI);
+  G4Tubs cylinder("Cylinder", 399., 401., 800., -M_PI * CLHEP::radian,
+                  2 * M_PI * CLHEP::radian);
   auto [bounds, thickness] =
```

Comment:
Does this have to change in order to test for the converters on the angles? (in Geant4Converters.cpp)

Reply:
I made it explicit for the reader, it doesn't have to change, `G4Tubs` is anyway doing a bit of its own thing, which is why I had to change the converter a little bit.